src/ui_utils.c

```diff
@@ -68,7 +68,7 @@
 	"filetype: %f      " \
 	"scope: %S")
 
-GeanyInterfacePrefs	interface_prefs;
+GEANY_EXPORT_SYMBOL GeanyInterfacePrefs interface_prefs;
```

Comment:
Why is this necessary?

Reply:
The unit test tests each of the modes, thus needs to modify `interface_prefs.openfiles_path_mode`

Reply:
Ah, I see.